apps/prairielearn/src/pages/instructorAssessmentQuestions/instructorAssessmentQuestions.html.ts

```diff
@@ -288,18 +289,9 @@ function AssessmentQuestionsTable({
                     : ''}
                 </td>
                 <td>
-                  ${question.other_assessments
-                    ? question.other_assessments.map((assessment) => {
-                        return html`${renderEjs(
-                          import.meta.url,
-                          "<%- include('../partials/assessment'); %>",
-                          {
-                            urlPrefix,
-                            assessment,
-                          },
-                        )}`;
-                      })
-                    : ''}
+                  ${question.other_assessments?.map((assessment) =>
+                    AssessmentBadge({ urlPrefix, assessment }),
+                  )}
```

Comment:
Is this perhaps needed?

```suggestion
                  ) ?? ''}
```

Reply:
See #9677.

Reply:
FWIW, we've been writing code under the assumption that this is the way we want to go.